public/index.html

```diff
@@ -26,17 +26,19 @@
     <link rel="mask-icon" href="%PUBLIC_URL%/safari-pinned-tab.svg" color="#5bbad5">
     <meta name="theme-color" content="#ffffff">
 
-    <link rel="preload" href="/fonts/ClanOffcPro-Bold.woff2" as="font" crossorigin="anonymous" />
-    <link rel="preload" href="/fonts/ClanOffcPro-Medium.woff2" as="font" crossorigin="anonymous" />
-    <link rel="preload" href="/fonts/ClanOffcPro-Book.woff2" as="font" crossorigin="anonymous" />
-    <link rel="preload" href="/fonts/material-icons.woff2" as="font" crossorigin="anonymous" />
+    <link rel="preload" href="/static/media/ClanOffcPro-Medium.woff2" as="font" crossorigin="anonymous" />
+    <link rel="preload" href="/static/media/ClanOffcPro-Book.woff2" as="font" crossorigin="anonymous">
+    <link rel="preload" href="/static/media/ClanOffcPro-Bold.woff2" as="font" crossorigin="anonymous" />
+    
+    <link rel="preload" href="/static/media/material-icons.woff2" as="font" crossorigin="anonymous" />
+    <link rel="preload" href="/static/media/cryptocoins.woff2" as="font" crossorigin="anonymous" />
 
     <script type="text/javascript">window._trackJs = { token: 'd378b378ded441aaa0f23b5403f62a53' };</script>
 
     <!-- <script type="text/javascript" src="https://cdn.trackjs.com/releases/current/tracker.js"></script> -->
 
-    <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.0.3/cookieconsent.min.css" />
-    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.13/css/all.css" integrity="sha384-DNOHZ68U8hZfKXOrtjWvjxusGo9WQnrNx2sqG0tfsghAvtVlRW3tvkXWZh58N9jp" crossorigin="anonymous">
+    <link as="style" rel="preload" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.0.3/cookieconsent.min.css" />
```

Comment:
Seems like we have cookieconsent.min.js twice

Reply:
First one is JS, another one is CSS.